content/guides/overview/choosing-testing-type.md

```diff
@@ -0,0 +1,138 @@
+
+</Alert>
+
+To learn more about end-to-end testing in Cypress, visit our guide on
+[Writing Your First End-to-end Test](/guides/getting-started/writing-your-first-end-to-end-test).
+
+## What is Component Testing?
+
+Modern web frameworks provide ways to write applications by breaking them into
+smaller logical units called components. Components can range from fairly small
+(like a button) to more complex (like a registration form).
+
+Because of their nature, components tend to be easily testable, which is where
+Cypress Component Testing comes into play.
+
+Component tests differ from end-to-end tests in that instead of visiting a URL
+to pull up an entire app, a component can be "mounted" and tested on its own.
+This allows you to focus on testing only the component's functionality and not
+worrying about other nuances with testing a component as part of the larger
+application.
+
+Typically, a component test is written by the developers working on the
+component. The code for the test lives alongside the component code, and it is
+common for tests to be coded simultaneously with the component, helping
+developers verify the required functionality while building it.
+
+One thing to consider, though, is even if all your component tests pass, it does
+not mean your app is functioning properly. Component tests do nothing to ensure
+that all the layers of your app are working well together. Therefore, a
+well-tested app has a combination of end-to-end and component tests, with each
+set of tests specializing in what they do best.
+
+<Alert type="success">
+
+### Benefits of component tests:
+
+- Easier to test components in isolation
+- Fast and reliable
+- Easy to set up specific scenarios in tests
+- Don't rely on any external system to run
+
+### Considerations for component tests:
+
+- Do not ensure overall app quality
+- Do not call into external APIs/Services
+- Usually written by developers working on the component
+
+### Common scenarios for component tests:
+
+- Testing a date picker works properly for a variety of scenarios
+- That a form shows and hides specific sections based on input
+- Testing components coming out of a design system
+- Testing logic not tied to a component (like unit tests!)
+
+</Alert>
+
+To learn more about component testing in Cypress, visit our guide on
+[Writing Your First Component Test](/guides/getting-started/writing-your-first-component-test).
+
+## Test Type Comparison
+
+| | End-to-end | Component | |-| |:-:| |:-:| |What's Tested | All app layers |
+Individual component | |Characteristics|Comprehensive, slower, more suspectable
+to flake| Specialized, quick, reliable | | Used For | Verifying app works as a
+cohesive whole | Testing functionality of individual component | | Written By |
+Developers, QA Team, SDETs | Developers, Designers | | CI Infrastructure | Often
+requires complex setup | None needed | | Initialization Command |
+`cy.visit(url)` | `mount(<MyComponent />)` |
```

Comment:
This isn't rendering properly: 

<img width="801" alt="CleanShot 2021-11-18 at 15 59 27@2x" src="https://user-images.githubusercontent.com/334337/142495831-65cff859-cec5-4c82-b977-e88e856a5b63.png">